Gemfile

```diff
@@ -13,4 +13,5 @@ gem 'scss_lint'
 
 group :jekyll_plugins do
   gem 'jekyll-multiple-languages-plugin'
+  gem 'jekyll-sitemap'
```

Comment:
@OleksSobol Probably you updated you branch not ptoblerly, you got files from previous PR https://github.com/howtohireme/ruby.fundamental/commit/94feecc55ed7ab5a636baa0a4cf6bc4be4cb43ca Check it please.